docs/src/contributing.md

```diff
@@ -12,14 +12,14 @@ You'll also want to check out the contributing [guidelines].
 
 ## 🏃‍♀️ Up and Running
 
-1. fork and clone this repository
+1. fork and clone the `rustwasm/wasm-pack` repository
```

Comment:
When reading at https://rustwasm.github.io/docs/wasm-pack/ it wasn't self-evident what "this repository" was